app/controllers/posts.rb

```diff
@@ -0,0 +1,64 @@
+get '/' do
+  redirect '/index'
+end
+
+get '/index' do
+  @questions = Question.all
+  @answer_counts = @questions.map{ |question| question.answers.count }
+  @scores = @questions.map{ |question| question.score }
+  erb :index
+end
+
+get '/questions' do
+  redirect '/index'
+end
+
+get '/questions/new' do
+  puts "current user: #{current_user.username}"
+  erb :"posts/new"
+end
+
+post '/questions' do
+  puts params.inspect
+  puts "user id: #{session[:user_id]}"
+  puts "current user: #{current_user}"
+  @question = current_user.questions.create(params)
+  redirect '/index'
+end
+
+post '/questions/:id/answers' do
+  Answer.create(params[:answer])
```

Comment:
i highly suggest having some type of control here in case the answer doesn't save. that would be no bueno.